src/QuickMarcEditor/utils.js

```diff
@@ -44,19 +44,35 @@ export const addNewRecord = (index, state) => {
   return records;
 };
 
+export const validateLeader = (prevLeader = '', leader = '') => {
+  const cutEditableBytes = (str) => (
+    [5, 8, 17, 18, 19].reduce((acc, byte, idx) => {
```

Comment:
is it ok that array is hardcoded?

Reply:
yes, other variants?

Reply:
to constant them?)

Reply:
constant is hardcode as well) IMO it's ok to have like it is, it's small and it shouldn't be reused